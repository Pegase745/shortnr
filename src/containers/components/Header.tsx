import * as React from 'react';
import { Image, Segment } from 'semantic-ui-react';

const style = {
  img: {
    paddingTop: '4em',
    width: '50%',
    maxWidth: 200,
  },
};

const StyledHeader: React.FC<{}> = () => (
  <Segment basic={true}>
    <Image
      as="img"
      style={style.img}
      src="assets/logo.png"
      centered={true}
      fluid={true}
    />
  </Segment>
);

export default React.memo<{}>(StyledHeader);
