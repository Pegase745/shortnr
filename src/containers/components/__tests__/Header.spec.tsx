import { shallow } from 'enzyme';
import * as React from 'react';

import Header from '../Header';

const defaultProps = {};

describe('Header.tsx', () => {
  it('should render', () => {
    const app = shallow(<Header {...defaultProps} />);

    expect(app).toMatchSnapshot();
  });
});
