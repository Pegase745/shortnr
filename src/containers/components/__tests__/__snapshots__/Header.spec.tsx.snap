// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.tsx should render 1`] = `
<Segment
  basic={true}
>
  <Image
    as="img"
    centered={true}
    fluid={true}
    src="assets/logo.png"
    style={
      Object {
        "maxWidth": 200,
        "paddingTop": "4em",
        "width": "50%",
      }
    }
    ui={true}
  />
</Segment>
`;
